frontend/src/routes/account/+page.svelte

```diff
@@ -1,30 +1,68 @@
-<div class="card flex-shrink-0 w-full lg:w-1/3 max-w-sm shadow-2xl bg-base-100">
-  <div class="card-body">
-    <div class="form-control">
-      <label class="label" for="email-input">
-        <span class="label-text">Email</span>
+<script>
```

Comment:
We should define this as typescript instead of javascript. Doing this and using typescript types rather than jsdoc I think will make a lot of the linting errors more clear.

```suggestion
<script lang="ts">
```